judgments/forms/search_forms.py

```diff
@@ -0,0 +1,114 @@
+from django import forms
+from django.forms import ValidationError
+from django.utils.translation import gettext as _
+from ds_caselaw_utils import courts as all_courts
+
+from judgments.utils import preprocess_query
+
+from .fields import DateRangeInputField
+from .widgets import CheckBoxSelectCourtWithYearRange
+
+
+def _get_choices_by_group(courts):
+    """
+    Given a list of court objects, construct a dictionary containing
+    {"key1": "value1"} for non-grouped courts, and {"court_group": {"key1": "value1"}}
+    for grouped courts. This allows us to nest choices within groups when rendering
+    a `MultipleChoiceField` field in Django.
+
+    e.g.
+    {
+        "court_group_1": {"key1": "value1"},
+        "key2": "value2"
```

Comment:
Would an explicit top level key for the unnested courts help?

Reply:
I am not sure what you mean?